File: Modulo2/semana05/aula02/ex5/calculadora.js
const soma = (a, b) => {
  const resultado = a + b;
  console.log('Resultado da soma', resultado);
};

const subtracao = (a, b) => {
  const resultado = a - b;
  console.log('Resultado da subtracao', resultado);
};

const multiplicacao = (a, b) => {
  const resultado = a * b;
  console.log('Resultado da multiplicacao', resultado);
};

const divisao = (a, b) => {
  const resultado = a / b;
  console.log('Resultado da divisao', resultado);
};

export default {
  soma,
  subtracao,
  multiplicacao,
  divisao,
}

File: Modulo 3/semana1/aula01/m03s01-aula1-main/src/tech/devinhouse/aula/Calculadora.java
package tech.devinhouse.aula;

public class Calculadora {

    public Float calcular(Float valor1, Float valor2, Operacao operacao) {
        Float resultado = null;
        switch (operacao) {
            case SOMA:
                resultado = valor1 + valor2;
                break;
            case SUBTRACAO:
                resultado = valor1 - valor2;
                break;
            case MULTIPLICACAO:
                resultado = valor1 * valor2;
                break;
            case DIVISAO:
                resultado = valor1 / valor2;
                break;
        }
        return resultado;
    }

}
